scripts/setup-check.sh

```
@@ -366,6 +366,31 @@ echo ""
+check_command "npm" "9+" "brew install node"
+
+# Check if markdownlint-cli2 is installed
+if [ -f "package.json" ]; then
+    if [ -d "node_modules" ] && npm list markdownlint-cli2 &> /dev/null; then
+        echo -e "${GREEN}✓${NC} markdownlint-cli2 installed"
+        echo -e "  Run: npm run lint:md"
+    else
+        echo -e "${YELLOW}⚠${NC}  markdownlint-cli2 not installed"
+        echo -e "  This is required for markdown linting in pre-commit hooks"
+        echo -e "  Install: npm install"
+        MISSING_TOOLS=true
```

Shellcheck (0.11.0)

[warning] MISSING_TOOLS appears unused. Verify use (or export if used externally).

(SC2034)